lib/web/scripts/oneoff/oneoff.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cdnBaseURL='{{.CDNBaseURL}}'
+teleportVersion='{{.TeleportVersion}}'
+tempDir=$({{.BinMktemp}} -d)
+OS=$({{.BinUname}} -s)
+ARCH=$({{.BinUname}} -m)
+teleportArgs='{{.TeleportArgs}}'
+
+function teleportTarballName(){    
+    if [[ ${OS} == "Darwin" ]]; then
+        echo teleport-${teleportVersion}-darwin-universal-bin.tar.gz
+        return 0
+    fi;
+
+    if [[ ${OS} != "Linux" ]]; then
+        echo "Only MacOS and Linux are supported." >&2
+        return 1
+    fi;
+
+    if [[ ${ARCH} == "armv7l" ]]; then echo "teleport-${teleportVersion}-linux-arm-bin.tar.gz"
+    elif [[ ${ARCH} == "aarch64" ]]; then echo "teleport-${teleportVersion}-linux-arm64-bin.tar.gz"
+    elif [[ ${ARCH} == "x86_64" ]]; then echo "teleport-${teleportVersion}-linux-amd64-bin.tar.gz"
+    elif [[ ${ARCH} == "i686" ]]; then echo "teleport-${teleportVersion}-linux-386-bin.tar.gz"
+    else
+        echo "Invalid Linux architecture ${ARCH}." >&2
+        return 1
+    fi;
+}
+
+function main() {
+    pushd $tempDir > /dev/null
+
+    tarballName=$(teleportTarballName)
+    curl --show-error --fail --location --remote-name ${cdnBaseURL}/${tarballName}
+    tar -xzf ${tarballName}
+
+    echo "> ./teleport/teleport ${teleportArgs}"
+    ./teleport/teleport ${teleportArgs}
```

Comment:
This looks dangerous. What if I use a query param of "; rm -rf /" ??

Reply:
Well, the whole `sudo bash -c "$(curl ...)"` is dangerous.
Because, if we assume a user can run a tampered query param, what is stopping the user to run a tampered command, something like this: `sudo bash -c "$(curl ... && echo rm -rf /)"`

If we want to provide some protection to the query param, we could use a new concept of command and args.
`.../oneoff?command=<command>&args=<b64 of json object containing the required arguments>`

Example of commands and their args:
a) `command=version`
b) `command=setupIntegrationAWSOIDCDeployServiceRoles&args=<b64 of json {"oidcRole": "SomeRole", "taskRole": "AnotherRole"}>`

When generating the script, each of those commands would translate into actual `teleport` commands.
Using the example above
a) would be `teleport version`
b) would be `teleport bootstrap integration awsoidc deploy-service-roles --oidcRole=SomeRole --taskRole=AnotherRole` (this command does not exist, but will be created in another PR)

What do you think?

Reply:
I'm going to reduce scope from this PR and remove the endpoint
We'll create specific endpoints per use case